app/src/main/java/com/github/fribourgsdp/radio/MusixmatchLyricsGetter.kt

```diff
@@ -124,10 +127,14 @@ object MusixmatchLyricsGetter : LyricsGetter {
                 if(parsedResponse == null){
                     future.completeExceptionally(Exception("Error parsing response"))
                 }
-                val trackList = parsedResponse
-                    ?.getJSONObject("message")
-                    ?.getJSONObject("body")
-                    ?.getJSONArray("track_list")
+                val trackList = try {
+                    parsedResponse
+                        ?.getJSONObject("message")
+                        ?.getJSONObject("body")
+                        ?.getJSONArray("track_list")
+                } catch (e : Exception){
+                    JSONArray()
```

Comment:
As discussed IRL, we can try and find the source of the error causing this exception to be thrown next week.

Reply:
Yes that was an emergency patch.